.bazelrc

```diff
@@ -4,9 +4,6 @@
 # bazel configurations for running tests under sanitizers.
 # Based on https://github.com/bazelment/trunk/blob/master/tools/bazel.rc
 
-# TODO: Remove once support is added, avoid MODULE.bazel creation for now
-common --enable_bzlmod=false
-
```

Comment:
Is there a way to support bazel modules on all platforms except windows, due to the file length disaster ?

Reply:
I don't think we can restrict it per platform in the BCR, so probably better we find the issues here in case users on windows try?

Reply:
I meant here, in this file:

somehow have this only for windows:

```
common --enable_bzlmod=false
```

so we can move on, with the windows limitation to be lifted later.

Reply:
oh, yea we definitely could, but now that things are passing i think we're probably fine? i think it's only likely to re-break if we update deps or something, so i don't think it would block normal development

Reply:
Works for me. Waiting on @ThomsonTan and/or @lalitb opinion.